src/containers/tutor-profile/coments-block/constants.js

```diff
@@ -0,0 +1,212 @@
+export const responseMock = {
+  count: 5,
+  items: [
+    {
+      _id: '6400f540307bdcc5da14aa5b',
+      comment: 'great',
+      rating: 5,
+      author: {
+        _id: '63f905d3237ccffcf95d88da',
+        role: ['student'],
+        firstName: 'Tart',
+        lastName: 'Drilling',
+        photo:
+          'https://media.npr.org/assets/img/2014/08/07/monkey-selfie_custom-7117031c832fc3607ee5b26b9d5b03d10a1deaca-s300-c85.webp',
+        email: 'tartdrilling@gmail.com',
+        categories: [],
+        totalReviews: 0,
```

Comment:
remove those fields.
there will be no totalReviews and averageRating

Reply:
fixed